### PR TITLE
Update README with packages and compat chart

### PR DESCRIPTION
**Pull request by GregSutcliffe**

### CODE REVIEW

**Comment by GregSutcliffe** [comment]
@domcleal are the rpm package names right?

File: README.md [hunk 1]
```diff
@@ -3,15 +3,29 @@
 
 A plugin to create images from Hostgroups
 
+## Installation
+
+Please see the Foreman wiki for appropriate instructions:
+
+* [Foreman: How to Install a Plugin](http://projects.theforeman.org/projects/foreman/wiki/How_to_Install_a_Plugin)
```
Comment (domcleal) [inline README.md:10]: Now in the manual.

**Comment by domcleal** [comment]
Yep, fine.

File: README.md (hunk 1 above)
Comment (GregSutcliffe) [inline README.md:10]: Updated thanks

**Comment by GregSutcliffe** [comment]
Merged as 5c745c5 and 1e48a06 thanks @domcleal for the review

**Pull request closed without merging**